packages/lesswrong/components/common/CookieBanner/geolocation.ts

```diff
@@ -1,7 +1,10 @@
 import { isServer } from "../../../lib/executionEnvironment";
 import { isEAForum } from "../../../lib/instanceSettings";
+import { DatabasePublicSetting } from "../../../lib/publicSettings";
 import { getBrowserLocalStorage } from "../../editor/localStorageHandlers";
 
+const ipApiKeySetting = new DatabasePublicSetting<string | null>('ipapi.apiKey', null);
```

Comment:
I have made this a public setting, if it gets abused we can do something more complicated where we get the ip address from the headers on the server and look it up there, but for now I'm just hoping it doesn't get abused. If it does the worst case is that the cookie banner shows in countries where it shouldn't